matchbox_socket/src/webrtc_socket/error.rs

```diff
@@ -11,6 +11,8 @@ pub enum SignallingError {
     StreamExhausted,
     #[error("Message received in unknown format")]
     UnknownFormat,
+    #[error("failed to establish initial connection")]
+    NoMoreAttempts(#[from] Box<SignallingError>),
```

Comment:
Since this is a public error, could the name perhaps make it more obvious that this is something that happens on connect?

`ConnectionFailed`?

Reply:
Changed to SignallingError::ConnectionFailed